client/src/components/ScorecardSummaryPanel.vue

```diff
@@ -63,13 +73,47 @@ export default defineComponent({
   },
   computed: {
     homeAgeComparison(): string | null {
-      return this.formatNumber(this.demographicDataState?.data?.averageHomeAge);
+      const homeAge = this.roundNumber(this.demographicDataState?.data?.averageHomeAge);
```

Comment:
nit can we check if this is null before rounding and just return?

same below for income

Reply:
Round number does that - it checks if it's null and just returns null in that case